### PR TITLE
update some package.json files

### PR DESCRIPTION
**Pull request by dong77**

### CODE REVIEW

**Comment by Brechtpd** [comment]
#2433

File: packages/hebao_v2/package.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.1",
-    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@nomiclabs/hardhat-waffle": "^1.0.0",
```
Comment (Brechtpd) [inline packages/hebao_v2/package.json:15]: What is the purpose of lowering the minimum version for some of these packages?

Comment (dong77) [reply packages/hebao_v2/package.json:15]: This PR is not ready. I'm sure how the above happened ：）

**Comment by dong77** [comment]
@Brechtpd can you pick up this PR and make sure it works for you?

**Review by Brechtpd — APPROVED** [review]

**Comment by Brechtpd** [comment]
All working for me now.

**Pull request merged**